src/stylesheets/global.css

```
@@ -2,7 +2,7 @@
 
 @import '~normalize.css';
 @import url('https://fonts.googleapis.com/css2?family=Merriweather:wght@300&display=swap');
-@import url('https://fonts.googleapis.com/css2?family=Lato:wght@400;700&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Lato:wght@400i,400;700&display=swap');
```

This doesn't appear to be working for me, and, disappointingly, it seems to fail silently.

I manually generated a new URL by going to [Google Fonts](https://fonts.google.com/specimen/Lato?query=lato#standard-styles) and manually selecting the variants we want, and I got the following URL, which seems to work correctly when I modify this line locally:

```suggestion
@import url('https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,400;0,700;1,400&display=swap');
```

I think I may have incorrectly suggested the wrong manual edit to you previously, since I now see that the `@import` statements from the legacy CSS files use `/css`, but this file uses `/css2`, which I'm assuming is Google's v2 of the API.

After I made the change locally with this URL, the italicized text finally looks like it has the same thickness as the regular text around it.

---

Weird, it was definitely working for me 🤔 This works too though!